packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -37,14 +37,14 @@ function InteractionLayer({
   const [creating, setCreating] = useState(false)
   const canvas = useRef()
 
-  useEffect(
-    function onDeleteMark() {
-      if (creating && !activeMark) {
-        setCreating(false)
-      }
-    },
-    [activeMark]
-  )
+  if (creating && !activeMark) {
+    setCreating(false)
+  }
```

Comment:
[minor][curiousity] Why was this removed from `useEffect`, by the way?

Reply:
This runs before the component renders, while `useEffect` would render, then run the side effect of the render. 

I think this is the explanation: https://beta.reactjs.org/apis/usestate#storing-information-from-previous-renders